src/synthesize.jl

```diff
@@ -202,7 +202,7 @@ You can provide either or both of:
 """
 function format_A_X(metallicity::Real=0.0, abundances::Dict=Dict();
```

Comment:
Do you mind adding a test case ensuring that the vector returned is the same regardless of of whether or not A(H) is specified?